Makefile

```
@@ -15,51 +14,79 @@ test-race: ##@quality validate race condition
+	sqlc generate
+
+.PHONY: smoke
+smoke: build ##@quality run smoke tests
+	./smoke.sh
+
+.PHONY: update-golden
+update-golden: ##@helper update golden files
+	go test ./cmd/cli -update
+
+.PHONY: release
+release: clean ##@application create a new release (usage: make release VERSION=v1.0.0)
```

checkmake (0.2.2)

[warning] Target body for "release" exceeds allowed length of 5 (8).

(maxbodylength)